tests/integration/cloudformation/test_cloudformation_stacks.py

```diff
@@ -139,3 +139,35 @@ def wait_stack_done():
     assert wait_until(wait_stack_done)
     stack_describe_response = cfn_client.describe_stacks(StackName=stack_id)["Stacks"][0]
     assert len(stack_describe_response["Outputs"]) == 2
+
+
+def test_stack_time_attributes(cfn_client, deploy_cfn_template):
+    bucket_name = "test"
```

Comment:
```suggestion
@pytest.mark.aws_validated
def test_stack_time_attributes(cfn_client, deploy_cfn_template):
    bucket_name = f"test-{short_uid()}"
```

Please test this against AWS and add the annotation when validated successfully :+1: 

The bucket_name not being unique leads to issues when deploying on AWS.